content/events-calendar/2024/AI-Hack-Day---Brisbane.json

```
@@ -14,7 +14,7 @@
   "calendarType": "Hack Days",
   "city": "Brisbane",
   "category": "Other",
-  "description": "​\n\nWhat is AI Hack Day\n\n?\n\nAI Hack Days are community run events for Developers or people who want to learn AI for free.\n\nThere will be something for everyone. Experienced AI developers can share ideas with other experienced developers. If you’re a beginner then there’s plenty to learn on the day.\n\nIf you then want to build something yourself, you'll need some of the Microsoft AI and/or Machine Learning (ML) tools already setup and installed. If you’ve never played with AI or ML before it would be good to do a little bit of learning before the day but it’s not a requirement as all experience levels are welcome.\n",
+  "description": "​Hack Days are community run events for Developers or people who want to learn AI for free.\n\nThere will be something for everyone. Experienced AI developers can share ideas with other experienced developers. If you’re a beginner then there’s plenty to learn on the day.\n\nIf you then want to build something yourself, you'll need some of the Microsoft AI and/or Machine Learning (ML) tools already setup and installed. If you’ve never played with AI or ML before it would be good to do a little bit of learning before the day but it’s not a requirement as all experience levels are welcome.\n",
```

I'm nit picking a bit but just to prerserve the style of the original article 🙂

```suggestion
  "description": "​What is AI Hack Day?\n\nHack Days are community run events for Developers or people who want to learn AI for free.\n\nThere will be something for everyone. Experienced AI developers can share ideas with other experienced developers. If you’re a beginner then there’s plenty to learn on the day.\n\nIf you then want to build something yourself, you'll need some of the Microsoft AI and/or Machine Learning (ML) tools already setup and installed. If you’ve never played with AI or ML before it would be good to do a little bit of learning before the day but it’s not a requirement as all experience levels are welcome.\n",
```